packages/core/src/types/authMachine.ts

```diff
@@ -33,6 +33,7 @@ export interface SignUpContext {
   login_mechanisms?: string[];
   intent?: string;
   authAttributes?: Record<string, any>;
+  challengeName?: string;
 }
```

Comment:
When I compiled my Vue, it had errors that the types weren't correct for some of these contexts. I had to add these so it would compile. Since they are all optional it should be OK.

Reply:
This is probably on me! But I prefer we don't add `challengeName` to other types, as this context is only necessary in signIn. Where have you found these errors?

Reply:
Resolving as discussed, we'll tackle this later.